docs/source/snippets.rst

```diff
@@ -0,0 +1,14 @@
+XRPL Code Snippets
+===================
+
+Code snippets to demonstrate basic usage of the xrpl-py library.
+
+.. toctree::
+   :maxdepth: 1
+
+   get_transaction
+   partial_payment
+   paths
+   reliable_transaction_submission
+   send_escrow
+   set_regular_key
```

Comment:
Also, is it possible to give these more human readable names?

Ex. 
* Look up a transaction on ledger
* Send a transaction and see it get validated
* Change your password by setting a regular key
* Set up an Escrow
* Find the best path to trade with
* Handle partial payments